packages/components/src/internal/components/chart/ChartBuilderModal.tsx

```diff
@@ -810,3 +872,135 @@ export const ChartBuilderModal: FC<ChartBuilderModalProps> = memo(({ actions, mo
+    const hidden = useMemo(() => {
+        const jsonType = fieldValues.x?.data?.jsonType || fieldValues.x?.data?.type;
+        return !fieldValues.x?.value || jsonType === 'date' || jsonType === 'time';
+    }, [fieldValues.x]);
+
+    const [loadingTrendlineOptions, setLoadingTrendlineOptions] = useState<boolean>(true);
+    const [asymptoteMin, setAsymptoteMin] = useState<string>('');
+    const [asymptoteMax, setAsymptoteMax] = useState<string>('');
+    useEffect(() => {
+        if (loadingTrendlineOptions && (!!fieldValues.trendlineAsymptoteMin || !!fieldValues.trendlineAsymptoteMax)) {
+            setAsymptoteMin(fieldValues.trendlineAsymptoteMin);
+            setAsymptoteMax(fieldValues.trendlineAsymptoteMax);
+            setLoadingTrendlineOptions(false);
+        }
+    }, [fieldValues, loadingTrendlineOptions]);
+
+    const onTrendlineAsymptoteMin = useCallback((event: any) => {
+        setAsymptoteMin(event.target.value);
+    }, []);
+
+    const onTrendlineAsymptoteMax = useCallback((event: any) => {
+        setAsymptoteMax(event.target.value);
+    }, []);
+
+    const applyTrendlineAsymptote = useCallback(() => {
+        onFieldChange('trendlineAsymptoteMin', asymptoteMin);
+        onFieldChange('trendlineAsymptoteMax', asymptoteMax);
+    }, [onFieldChange, asymptoteMin, asymptoteMax]);
+
+    const onTrendlineFieldChange = useCallback(
+        (key: string, _: any, selectedOption: SelectInputOption) => {
+            setAsymptoteMin('');
+            onFieldChange('trendlineAsymptoteMin', undefined);
+            setAsymptoteMax('');
+            onFieldChange('trendlineAsymptoteMax', undefined);
+            onFieldChange(key, selectedOption);
+        },
+        [onFieldChange]
+    );
+
+    useEffect(() => {
+        // if the x-axis measure has been removed or the option is hidden, clear the trendline type
+        if (hidden && fieldValues.trendlineType?.value) {
+            onTrendlineFieldChange('trendlineType', undefined, undefined);
+        }
+    }, [hidden, fieldValues, onTrendlineFieldChange]);
+
+    const trendlineOptions = useMemo(() => {
+        return TRENDLINE_OPTIONS.filter(option => {
+            return !option.schemaPrefix || schemaQuery.schemaName.startsWith(option.schemaPrefix);
+        });
+    }, [TRENDLINE_OPTIONS, schemaQuery.schemaName]);
+
+    if (hidden) return null;
+
+    return (
+        <div>
+            <label>
+                Trendline{' '}
+                <LabelOverlay placement="bottom" >
+                    <table>
+                        <tbody>
+                            {trendlineOptions
+                                .filter(option => option.equation)
+                                .map(option => (
+                                    <tr key={option.value}>
+                                        <td>
+                                            <b>{option.label}</b>
```

Comment:
The `<b>` tag is deprecated. If we must use a tag to get the styling we want, we should use `strong`, however like the equation below we could use a CSS class. The upside to using a CSS class over a `<strong>` element or similar is that our tests now have something semantic to query for e.g. `Locator.byCss('.equation-label')`.

```suggestion
                                            <strong>{option.label}</strong>
```

Reply:
done